packages/components/src/components/SquigglePlayground/index.tsx

```diff
@@ -196,15 +202,27 @@ export const SquigglePlayground: React.FC<PlaygroundProps> = (props) => {
   const leftPanelRef = useRef<HTMLDivElement | null>(null);
 
   const withEditor = (
-    <div className="flex mt-2">
+    <div className="mt-2 flex flex-row">
+      <ResizableBox
+        style={{ height: "100%" }}
```

Comment:
This is the same as `h-full`, which is set in `className` below, so probably not necessary?

Reply:
Good point!